src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/NumberToLocalizedStringTransformerTest.php

```diff
@@ -65,7 +65,7 @@ public static function provideTransformations()
     public function testTransform($from, $to, $locale)
     {
         // Since we test against other locales, we need the full implementation
-        IntlTestHelper::requireFullIntl($this, false);
+        IntlTestHelper::requireFullIntl($this, null);
```

Comment:
I suggest omitting `null` entirely as it is optional (as done in some of the impacted files btw).